src/components/progress-tracker/progress-tracker.style.ts

```diff
@@ -111,16 +111,16 @@ const StyledValuesLabel = styled.span<
 `;
 
 const InnerBar = styled.span<
-  Pick<ProgressTrackerProps, "progress" | "size" | "length" | "error">
+  Required<Pick<ProgressTrackerProps, "progress" | "size" | "error">>
 >`
-  ${({ progress, size = "medium", length, error, theme }) => css`
+  ${({ progress, size = "medium", error, theme }) => css`
```

Comment:
Apologies I wasn't clear here, but we don't need to default the `size` prop here anymore (it's defaulted in the component) now we've removed the potential for it to be undefined